src/System.Text.Json/tests/Serialization/BlogPost.cs

```diff
@@ -0,0 +1,822 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Net;
+
+namespace System.Text.Json.Serialization.Tests.Schemas.BlogPost
+{ 
+    public class Post
```

Comment:
Talked with @layomia about this POCO, he used this generator https://app.quicktype.io/#l=cs&r=json2csharp which doc : https://github.com/quicktype/quicktype/blob/master/FAQ.md says: 
![image](https://user-images.githubusercontent.com/45579687/61914427-6c609400-aef5-11e9-9f24-1f3121bb728d.png) 
so I assume there is no ownership/distribution/licensing issue with using this POCO.  
cc @richlander

Reply:
I learned recently from @terrajobst, there is a way to generate .NET POCO from JSON built-into Visual Studio. I don't know how accurate it is, but we could consider using that instead.

![image](https://user-images.githubusercontent.com/6527137/61915027-8f8c4300-aef7-11e9-8035-119ac0233149.png)


Doing so from the json file in https://github.com/dotnet/corefx/pull/39094 (https://github.com/dotnet/corefx/blob/c3053820eab4bf0fc9606dc6088c580ac3197201/src/System.Text.Json/tests/Resources/Payloads/BlogPost.json) generates the following POCO types:

https://gist.github.com/ahsonkhan/c92fa6f825849d7c8e1f8b3027754a50

The question is, where did the original JSON file itself come from?

Reply:
Also, for our particular use case here, I don't know if we need to use "real world" JSON blobs. We could generate our own sample .NET object types and use those to serialize large JSON payloads.

Reply:
> I learned recently from @terrajobst, there is a way to generate .NET POCO from JSON built-into Visual Studio. I don't know how accurate it is, but we could consider using that instead.

I will consider using that

> The question is, where did the original JSON file itself come from

Not sure about that, anyways we are not using the json is that matter?

> Also, for our particular use case here of improving test coverage for large JSON, I don't know if we need to use "real world" JSON blobs. We could generate our own sample .NET object types and use those to serialize large JSON payloads.

Of course we can generate a large object for testing, that was easier way, but that would have some pattern/rule/loop which wouldn't be so close to or cover real life large objects scenario. So i preferred to use a POCO close to real